src/components/Albums/Albums.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const AlbumList = ({ name, albums }) => (
```

Comment:
I would pass through artist and your handleNoImage function that you're passing through from AlbumContainer instead of name.